integration-tests/tests/integrations.rs

```diff
@@ -3585,6 +3435,145 @@ mod slow {
 
         assert_eq!(bala_sim, bala_syn);
     }
+    async fn load_wallet_from_data_and_assert(
+        data: &[u8],
+        expected_balance: u64,
+        num_addresses: usize,
+    ) {
+        let config = zingoconfig::ZingoConfig::build(ChainType::Testnet)
+            .set_lightwalletd_uri(
+                ("https://zcash.mysideoftheweb.com:19067")
```

Comment:
Ooh, working testnet server?

Reply:
Why do we need a lwd uri at all? Testing wallet reading and writing shouldn't need a network connection at all

Reply:
only for testing that it sends